src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig

```diff
@@ -295,114 +589,220 @@ td.main, td.menu {
-    height: 30px;
-    display: none;
+#twig-dump .status-warning {
+    background: rgba(240, 181, 24, 0.3);
 }
-.sf-profiler-timeline .legends {
+#twig-dump .status-ok {
+    background: rgba(100, 189, 99, 0.2);
+}
+
+table.logs .metadata {
+    display: block;
     font-size: 12px;
-    line-height: 1.5em;
+    padding-top: 4px;
 }
-.sf-profiler-timeline .legends span {
-    border-left-width: 10px;
-    border-left-style: solid;
-    padding: 0 10px 0 5px;
+
+.status-ok {
+    background: rgba(100, 189, 99, 0.2);
+    background: rgba(94, 151, 110, 0.2);
 }
-.sf-profiler-timeline canvas {
-    border: 1px solid #999;
-    border-width: 1px 0;
+.status-warning {
+    background: rgba(240, 181, 24, 0.2);
+}
+.status-error {
+    background: rgba(204, 0, 0, 0.15);
+    background: rgba(176, 65, 62, 0.1);
+}
+.status-ok td,
+.status-warning td,
+.status-error td
+ {
+    background: transparent;
+}
+
+#search-results td {
+    vertical-align: middle;
 }
-.collapsed-menu-parents #resume,
-.collapsed-menu-parents #collector-content {
-    margin-left: 60px !important;
+#search-results .status-warning td.bold  {
+    color: #BC8034;
 }
-.collapsed-menu {
-    width: 60px !important;
+#search-results .status-error td.bold  {
+    color: #B0413E;
 }
-.collapsed-menu span.label > :not(.icon) {
-    display: none !important;
+
+.sql-queries th span {
+    color: #999;
 }
-.collapsed-menu #minimizePanelIcon {
-    transform: rotate(180deg);
+.sql-section
+.sql-actions {
+    margin-top: 4px;
 }
+.sql-actions a {
```

Comment:
should it really be called ``sql-actions`` ? What makes it specific to SQL ? Other panels may want to have actions too